docs/installation.rst

```diff
@@ -203,6 +203,7 @@ Install the dependencies with development headers::
     libacl1-dev libacl1 \
     libssl-dev \
     liblz4-dev libzstd-dev libxxhash-dev \
+    libffi-dev \
```

Comment:
ok, good.

Please add after line 170:

- libffi (required for argon2-cffi)

Reply:
In your special case, there is no pre-built wheel for `argon2-cffi-bindings`, so it needs to compile it and for that it needs the headers.

https://pypi.org/project/argon2-cffi-bindings/#files